File: Easy/Python/2475. Number of Unequal Triplets in Array.py
class Solution(object):
    def unequalTriplets(self, nums):
        count = 0
        n = len(nums)

        for i in range(0, n - 2, 1):
            a = nums[i]
            for j in range(i + 1, n - 1, 1):
                b = nums[j]
                for k in range(i + 2, n, 1):
                    c = nums[k]
                    if i < j and j < k:
                        if a != b and b != c and a != c:
                            count += 1

        return count

File: Easy/JavaScript/2475. Number of Unequal Triplets in Array.js
var unequalTriplets = function(nums) {
    let count = 0;
    let n = nums.length;

    for(let i = 0; i < n - 2; i++){
        let a = nums[i];
        for(let j = i + 1; j < n - 1; j++){
            let b = nums[j];
            for(let k = i + 2; k < n; k++){
                let c = nums[k];
                if(i < j && j < k){
                    if(a != b && b != c && a != c)
                        count++;
                }
            }
        }
    }

    return count;
};
